common/modal.js

```diff
@@ -25,6 +25,30 @@
             $uibModalInstance.close();
         }
     }])
+    .controller('LoginDialogController', ['$uibModalInstance', 'params' , '$sce', function LoginDialogController($uibModalInstance, params, $sce) {
+        var vm = this;
+        params.login_url = $sce.trustAsResourceUrl(params.login_url);
+        vm.params = params;
+
+        vm.openWindow = function() {
+
+            var x = window.innerWidth/2 - 800/2;
+            var y = window.innerHeight/2 - 600/2;
+
+            window.open(params.login_url, '_blank','width=800,height=600,left=' + x + ',top=' + y);
```

Comment:
This opens the window in the bottom right corner of my display no matter what size my browser is.